docs/docs/get-started.md

```diff
@@ -48,8 +50,10 @@ Recce expects two sets of dbt [artifacts](https://docs.getdbt.com/reference/arti
 - `target-base/` - dbt artifacts for to be used as the base for the comparison e.g. production
 - `target/` - dbt artifacts for your development branch
 
-#### Generate artifacts for the `base` environment
+#### Prepare artifacts for the `base` environment
+For most data warehouses, you can download the artifacts from your main branch. You don't need to re-run the whole production in your local. 
```

Comment:
I feel something like "download the artifacts from your base environment (normally it's the codebase of your main branch)" is closer.